probe-rs/src/architecture/riscv/mod.rs

```diff
@@ -539,6 +539,12 @@ impl<'probe> CoreInterface for Riscv32<'probe> {
 
         Ok(breakpoints)
     }
+
+    fn fpu_support(&mut self) -> Result<bool, crate::error::Error> {
+        Err(crate::error::Error::Other(anyhow::anyhow!(
```

Comment:
I think for these I'd return `false` and put a TODO versus error.  That way people can write code that depends on this and the behavior will match today (we don't have FPU support quite yet)

Reply:
Actually here I think error is correct :) This is completely new API surface so people would have to adapt to it anyways. Returning false would give a false image of the actual capabilities :)